fetch2/src/main/java/com/tonyodev/fetch2/downloader/FileDownloader.kt

```diff
@@ -33,4 +35,35 @@ interface FileDownloader : Runnable {
 
     }
 
-}
\ No newline at end of file
+}
+
+abstract class AbsFileDownloader(
```

Comment:
Is this class needed?

Reply:
Maybe, just added it for wrap shared checking for file-downloader which I think it is consistent